packages/atomic/src/components/commerce/atomic-commerce-breadbox/atomic-commerce-breadbox.tsx

```diff
@@ -251,7 +266,7 @@ export class AtomicCommerceBreadbox
             i18n: this.bindings.i18n,
             field: field,
             manualRanges: [],
-            formatter: defaultNumberFormatter,
+            formatter: this.getNumberFormatter(field),
```

Comment:
This is needed to ensure the breadcrumb for price values is formatted in the same way as the in the facet